package.json

```diff
@@ -28,33 +28,18 @@
     "ipfs",
     "install"
   ],
-  "author": "Juan Benet <juan@benet.ai> (http://juan.benet.ai/)",
+  "author": "Mark Robert Henderson <mark@mrh.io> (https://mrh.io/)",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/ipfs/npm-go-ipfs-dep/issues"
+    "url": "https://github.com/ipfs/npm-rust-ipfs-dep/issues"
   },
-  "homepage": "https://github.com/ipfs/npm-go-ipfs-dep",
+  "homepage": "https://github.com/ipfs/npm-rust-ipfs-dep",
   "devDependencies": {
     "aegir": "^18.2.0",
     "execa": "^1.0.0",
     "rimraf": "^2.6.3",
     "standard": "^12.0.1",
     "tap-spec": "^5.0.0",
     "tape": "^4.9.1"
-  },
-  "contributors": [
-    "Alan Shaw <alan.shaw@protocol.ai>",
-    "Alex Potsides <alex@achingbrain.net>",
-    "David Dias <daviddias.p@gmail.com>",
-    "Edward Silverton <e.silverton@gmail.com>",
-    "Friedel Ziegelmayer <dignifiedquire@gmail.com>",
-    "Haad <haadcode@users.noreply.github.com>",
-    "Harlan T Wood <harlantwood@users.noreply.github.com>",
-    "Henrique Dias <hacdias@gmail.com>",
-    "Oli Evans <oli@tableflip.io>",
-    "Scott Hardy <scott.the.hardy@gmail.com>",
-    "Stephen Whitmore <stephen.whitmore@gmail.com>",
-    "haad <haad@headbanggames.com>",
-    "olizilla <olizilla@users.noreply.github.com>"
-  ]
+  }
```

Comment:
Not sure if this is the right move, we are still building on all of their contributions, yes? Or are these indexed in npm in some way it might be beneficial for the contributors not have a pre-alpha rust-ipfs dep linked to their contribs?

Reply:
Contributors restored